### PR TITLE
feat(vivo new skin): adjust font weight in some components for Vivo Type font

### PR DESCRIPTION
**Pull request by atabel**
https://jira.tid.es/browse/WEB-822
https://www.figma.com/file/PApssATUkePZnIOB4GmAeI/branch/XrVdts3xEHrQAFC3ztaQgE/%F0%9F%94%B9-%E2%80%8E%E2%80%8ETokens-definition?type=design&node-id=2190-1642&t=BduCRWW3Fl7AnW3g-0

https://jira.tid.es/browse/OBVIVO-1848

### CODE REVIEW

File: src/card.tsx [hunk 1]
```diff
@@ -211,15 +211,17 @@ const transitions: Record<VideoState, Partial<Record<VideoAction, VideoState>>>
 const videoReducer = (state: VideoState, action: VideoAction): VideoState =>
     transitions[state][action] || state;
 
+const VideoSpinner = ({size, color}: IconProps) => <Spinner size={size} color={color} delay="0" />;
+
 const getVideoActionIcon = (state: VideoState) => {
     switch (state) {
         case 'playing':
+        case 'loading':
```
Comment (atabel) [inline src/card.tsx:219]: Unrelated change
Videos in cards have autoplay, so I think it makes sense that the initial icon is IconPausedFilled, otherwise we'd have a flash of a few ms showing IconPlayFilled

File: src/card.tsx [hunk 2]
```diff
@@ -211,15 +211,17 @@ const transitions: Record<VideoState, Partial<Record<VideoAction, VideoState>>>
 const videoReducer = (state: VideoState, action: VideoAction): VideoState =>
     transitions[state][action] || state;
 
+const VideoSpinner = ({size, color}: IconProps) => <Spinner size={size} color={color} delay="0" />;
```
Comment (atabel) [inline src/card.tsx:214]: remove the default spinner delay

File: .storybook/css/vivo-font.css
Comment (atabel) [inline .storybook/css/vivo-font.css:1]: added support for Vivo Type font in Storybook and Playroom

File: src/card.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/card.tsx:219]: If the video doesn't load fast enough, we would have a pause icon, followed by a spinner, followed by a pause icon again.

In my opinion, that is a bit weird

File: src/skins/types/index.tsx
Comment (yceballost) [inline src/skins/types/index.tsx:1]: just curious, why don't we include it in skin files?

**Review by pladaria — APPROVED** [review]

**Comment by aweell** [comment]
It is possible to use the weight tokens in playroom? In the same way we use skinVars.colors or skinVars.borderRadii?

**Review by yceballost — APPROVED** [review]

File: src/card.tsx (hunk 1 above)
Comment (atabel) [reply src/card.tsx:219]: I think it's weirder now in the most common case

**Comment by atabel** [comment]
> It is possible to use the weight tokens in playroom? In the same way we use skinVars.colors or skinVars.borderRadii?

Hmmm... I could expose them, but I think these tokens shouldn't be public, they only apply to internal details of mistica components, I think they shouldn't be used outside.

File: src/text.tsx [hunk 3]
```diff
@@ -148,22 +148,36 @@ interface LightProps extends TextPresetProps {
...
-type LightRegularMediumProps = LightProps | RegularProps | MediumProps;
+interface RestrictedWeightTextProps<T> extends TextPresetProps {
+    weight: T;
+    light?: undefined;
+    regular?: undefined;
+    medium?: undefined;
+}
+
+type RegularMediumProps = RegularProps | MediumProps | RestrictedWeightTextProps<'regular' | 'medium'>;
+type LightRegularMediumProps =
```
[the remark this answers is not in the document]
Comment (atabel) [reply src/text.tsx:176]: yes, I'll give it a try

[the remark this answers is not in the document]
Comment (atabel) [reply src/text.tsx]: done

**Review by aweell — APPROVED** [review]

**Review by marcoskolodny — APPROVED** [review]
